.github/workflows/automerge.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
I think you should put the "if" condition here immediately under `build:` - that way the job will show as 'skipped' for PRs it doesn't apply to. Otherwise it'll look like it always runs.